test/units/TEST-87-AUX-UTILS-VM.bootctl.sh

```diff
@@ -297,7 +303,11 @@ EOF
     for var in PK KEK db; do
         test -f "$(bootctl --print-esp-path)/loader/keys/auto/$var.auth"
     done
-    bootctl remove
+
+    if [ -d /tmp/auto.bak ]; then
+        rm -rf "$(bootctl --print-esp-path)/loader/keys/auto"
+        mv /tmp/auto.bak "$(bootctl --print-esp-path)/loader/keys/auto"
+    fi
```

Comment:
Maybe better to do that in a cleanup function with `trap`?

Reply:
forgot about that one